README.md

```diff
@@ -1,88 +1,129 @@
 
-## Features/APIs
+Collect and report usage data for your app. Our default plugin
+communicates with Amazon Pinpoint.
 
-- [**Analytics**](https://docs.amplify.aws/lib/analytics/getting-started?platform=android): Easily collect and report usage data for your app.
-- [**API**](https://docs.amplify.aws/lib/graphqlapi/getting-started?platform=android): An interface for interacting with REST and GraphQL endpoints. Provides AWS Signature Version 4 signing.
-- [**DataStore**](https://docs.amplify.aws/lib/datastore/getting-started?platform=android): Provides a persistent on-device storage repository for you to write, read, and observe changes to data. Data is sycnhronized to the cloud as well as across devices.
-- [**Storage**](https://docs.amplify.aws/lib/storage/getting-started?platform=android): Manage user content for your app, storing it in public, protected or private buckets.
+### [Storage](https://docs.amplify.aws/lib/storage/getting-started/q/platform/android):
+
+Store and retrieve files in the Cloud. We use S3 by default.
 
 ## Platform Support
 
-Amplify SDK supports Android API level 16 (Android 4.1) and above.
+The Amplify Framework supports Android API level 16 (Android 4.1) and above.
 
-## Installation
+## Using Amplify from Your App
 
-### Using Gradle
+### Specifying Gradle Dependencies
 
-To use Amplify, specify which modules you want to use inside of your app's `build.gradle` dependencies section:
+To begin, include Amplify from your `app` module's `build.gradle`
+dependencies section:
 
 ```gradle
 dependencies {
-    // Specify only the modules that the app will use
+    // Only specify modules that provide functionality your app will use
     implementation 'com.amplifyframework:aws-datastore:0.10.0'
     implementation 'com.amplifyframework:aws-api:0.10.0'
     implementation 'com.amplifyframework:aws-storage-s3:0.10.0'
     implementation 'com.amplifyframework:aws-analytics-pinpoint:0.10.0'
 }
 ```
 
-### Local Publishing of Artifacts
-
-You can manually install the library by cloning this repo and publishing the Android modules to the local maven repository.
+### Java 8 Compatibility
 
-Execute following commands from the project root:
+Amplify Android uses Java 8 features. Please add a `compileOptions`
+block inside your app's `build.gradle`, as below:
 
-```
-./gradlew publishToMavenLocal
+```gradle
+android {
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
+}
 ```
 
-Locally published artifacts can be accessed by specifying `mavenLocal()` inside the app's `build.gradle` file:
+### AWS Mobile Client
 
-``` gradle
-buildscript {
-    repositories {
-        mavenLocal() // This should ideally appear before other repositories
-    }
-    dependencies {
-        classpath 'com.android.tools.build:gradle:3.6.1'
+The default plugins for Amplify Android rely on the AWS Mobile Client to
```

Comment:
AWS Mobile Client is a somewhat drab, generic name whose Google results are confusing. Perhaps we can link to something for someone to read up? Our SDK auth guide?